classes/Utils.php

```diff
@@ -1263,6 +1248,42 @@ public function get_raw_course_price( $course_id = 0 ) {
 		return (object) $prices;
 	}
 
+	/**
+	 * Get prices with tax info
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param int|float $regular_price regular price.
+	 * @param int|float $sale_price sale price.
+	 *
+	 * @return object
+	 */
+	public function get_prices_with_tax_info( $regular_price, $sale_price = null ) {
+
+		$display_price       = $sale_price ? $sale_price : $regular_price;
+		$show_price_with_tax = Tax::show_price_with_tax();
+		$user_logged_in      = is_user_logged_in();
+
+		$tax_amount = 0;
+		$tax_rate   = 0;
+		if ( $show_price_with_tax && is_numeric( $display_price ) && ! Tax::is_tax_included_in_price() ) {
```

Comment:
`is_numeric( $display_price )`
Need to make sure there is no comma (**10,100.10**) in the price otherwise is_numeric will not work

Reply:
We are only accepting `int|flat` value in param without formatted value.